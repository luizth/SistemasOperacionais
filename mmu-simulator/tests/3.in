13
11 39
4 5
9 26
10 108
4 116
2 110
2 99
9 16
10 108
1 98
6 49
2 25
7 9
9 123
5 53
1 98
3 36
4 34
4 116
2 45
11 55
12 57
4 121
9 78
1 98
10 30
8 112
6 18
3 21
10 58
7 83
4 121
6 86
9 92
4 119
11 27
6 90
5 80
1 89
11 0
0 122
8 43
0 122
0 15
10 1
10 108
6 18
5 53
4 121
5 53
6 41
12 64
1 73
8 96
9 48
8 43
4 17
7 75
9 123
3 37
0 69
7 9
9 16
8 47
0 69
2 99
11 55
10 124
3 46
10 58
8 22
12 91
4 51
0 67
3 36
10 1
5 24
0 122
6 86
7 8
9 104
10 118
1 89
7 107
4 77
6 82
6 90
11 33
3 50
4 77
6 18
2 88
10 124
9 48
3 21
1 101
9 123
0 117
2 56
6 81
8 112
0 122
12 97
9 48
12 6
7 95
10 118
7 59
3 40
6 100
3 50
7 95
2 99
11 33
12 6
9 78
2 25
6 82
8 84
12 93
11 29
6 82
11 39
1 11
11 29
1 101
6 100
4 34
1 79
9 26
11 35
10 108
5 60
3 7
4 119
6 41
11 33
3 44
1 101
8 112
9 48
2 105
9 123
7 75
2 25
8 85
3 13
0 122
9 104
4 116
5 115
8 84
0 68
8 112
4 121
12 20
3 40
6 90
12 91
12 106
2 45
12 57
7 120
12 91
8 85
12 106
3 40
1 28
1 14
10 58
1 28
7 59
12 93
9 26
1 101
12 76
1 89
10 108
5 115
10 124
12 57
2 45
6 49
11 0
3 74
3 74
8 22
5 54
10 124
11 35
7 75
1 89
1 73
10 1
5 60
4 34
5 65
7 8
0 61
10 58
0 122
12 97
1 79
4 5
1 73
8 112
1 98
10 1
10 108
12 91
7 120
6 18
11 33
12 91
0 2
7 59
0 15
0 15
1 14
5 54
10 118
11 29
11 29
7 95
9 26
6 86
8 126
3 72
12 91
3 13
0 117
8 22
7 95
7 120
5 53
11 55
6 82
12 76
6 32
10 118
3 7
3 114
12 93
9 123
4 34
7 83
7 8
12 20
12 76
3 74
0 122
9 92
10 23
1 101
1 11
11 94
5 24
0 68
5 115
11 29
2 45
10 118
4 17
