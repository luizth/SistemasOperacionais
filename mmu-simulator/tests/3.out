Process 11 accessed page 39 - frame 0
Process 4 accessed page 5 - frame 1
Process 9 accessed page 26 - frame 2
Process 10 accessed page 108 - frame 3
Process 4 accessed page 116 - frame 4
Process 2 accessed page 110 - frame 5
Process 2 accessed page 99 - frame 6
Process 9 accessed page 16 - frame 7
Page 108 in memory (frame 3)!
Process 10 accessed page 108 - frame 3
Memory is full! Frame 0 was replaced (page 39 - pid 11)
Process 1 accessed page 98 - frame 0
Memory is full! Frame 1 was replaced (page 5 - pid 4)
Process 6 accessed page 49 - frame 1
Memory is full! Frame 2 was replaced (page 26 - pid 9)
Process 2 accessed page 25 - frame 2
Memory is full! Frame 3 was replaced (page 108 - pid 10)
Process 7 accessed page 9 - frame 3
Memory is full! Frame 4 was replaced (page 116 - pid 4)
Process 9 accessed page 123 - frame 4
Memory is full! Frame 5 was replaced (page 110 - pid 2)
Process 5 accessed page 53 - frame 5
Page 98 in memory (frame 0)!
Process 1 accessed page 98 - frame 0
Memory is full! Frame 6 was replaced (page 99 - pid 2)
Process 3 accessed page 36 - frame 6
Memory is full! Frame 7 was replaced (page 16 - pid 9)
Process 4 accessed page 34 - frame 7
Memory is full! Frame 0 was replaced (page 98 - pid 1)
Process 4 accessed page 116 - frame 0
Memory is full! Frame 1 was replaced (page 49 - pid 6)
Process 2 accessed page 45 - frame 1
Memory is full! Frame 2 was replaced (page 25 - pid 2)
Process 11 accessed page 55 - frame 2
Memory is full! Frame 3 was replaced (page 9 - pid 7)
Process 12 accessed page 57 - frame 3
Memory is full! Frame 4 was replaced (page 123 - pid 9)
Process 4 accessed page 121 - frame 4
Memory is full! Frame 5 was replaced (page 53 - pid 5)
Process 9 accessed page 78 - frame 5
Memory is full! Frame 6 was replaced (page 36 - pid 3)
Process 1 accessed page 98 - frame 6
Memory is full! Frame 7 was replaced (page 34 - pid 4)
Process 10 accessed page 30 - frame 7
Memory is full! Frame 0 was replaced (page 116 - pid 4)
Process 8 accessed page 112 - frame 0
Memory is full! Frame 1 was replaced (page 45 - pid 2)
Process 6 accessed page 18 - frame 1
Memory is full! Frame 2 was replaced (page 55 - pid 11)
Process 3 accessed page 21 - frame 2
Memory is full! Frame 3 was replaced (page 57 - pid 12)
Process 10 accessed page 58 - frame 3
Memory is full! Frame 4 was replaced (page 121 - pid 4)
Process 7 accessed page 83 - frame 4
Memory is full! Frame 5 was replaced (page 78 - pid 9)
Process 4 accessed page 121 - frame 5
Memory is full! Frame 6 was replaced (page 98 - pid 1)
Process 6 accessed page 86 - frame 6
Memory is full! Frame 7 was replaced (page 30 - pid 10)
Process 9 accessed page 92 - frame 7
Memory is full! Frame 0 was replaced (page 112 - pid 8)
Process 4 accessed page 119 - frame 0
Memory is full! Frame 1 was replaced (page 18 - pid 6)
Process 11 accessed page 27 - frame 1
Memory is full! Frame 2 was replaced (page 21 - pid 3)
Process 6 accessed page 90 - frame 2
Memory is full! Frame 3 was replaced (page 58 - pid 10)
Process 5 accessed page 80 - frame 3
Memory is full! Frame 4 was replaced (page 83 - pid 7)
Process 1 accessed page 89 - frame 4
Memory is full! Frame 5 was replaced (page 121 - pid 4)
Process 11 accessed page 0 - frame 5
Memory is full! Frame 6 was replaced (page 86 - pid 6)
Process 0 accessed page 122 - frame 6
Memory is full! Frame 7 was replaced (page 92 - pid 9)
Process 8 accessed page 43 - frame 7
Page 122 in memory (frame 6)!
Process 0 accessed page 122 - frame 6
Memory is full! Frame 0 was replaced (page 119 - pid 4)
Process 0 accessed page 15 - frame 0
Memory is full! Frame 1 was replaced (page 27 - pid 11)
Process 10 accessed page 1 - frame 1
Memory is full! Frame 2 was replaced (page 90 - pid 6)
Process 10 accessed page 108 - frame 2
Memory is full! Frame 3 was replaced (page 80 - pid 5)
Process 6 accessed page 18 - frame 3
Memory is full! Frame 4 was replaced (page 89 - pid 1)
Process 5 accessed page 53 - frame 4
Memory is full! Frame 5 was replaced (page 0 - pid 11)
Process 4 accessed page 121 - frame 5
Page 53 in memory (frame 4)!
Process 5 accessed page 53 - frame 4
Memory is full! Frame 6 was replaced (page 122 - pid 0)
Process 6 accessed page 41 - frame 6
Memory is full! Frame 7 was replaced (page 43 - pid 8)
Process 12 accessed page 64 - frame 7
Memory is full! Frame 0 was replaced (page 15 - pid 0)
Process 1 accessed page 73 - frame 0
Memory is full! Frame 1 was replaced (page 1 - pid 10)
Process 8 accessed page 96 - frame 1
Memory is full! Frame 2 was replaced (page 108 - pid 10)
Process 9 accessed page 48 - frame 2
Memory is full! Frame 3 was replaced (page 18 - pid 6)
Process 8 accessed page 43 - frame 3
Memory is full! Frame 4 was replaced (page 53 - pid 5)
Process 4 accessed page 17 - frame 4
Memory is full! Frame 5 was replaced (page 121 - pid 4)
Process 7 accessed page 75 - frame 5
Memory is full! Frame 6 was replaced (page 41 - pid 6)
Process 9 accessed page 123 - frame 6
Memory is full! Frame 7 was replaced (page 64 - pid 12)
Process 3 accessed page 37 - frame 7
Memory is full! Frame 0 was replaced (page 73 - pid 1)
Process 0 accessed page 69 - frame 0
Memory is full! Frame 1 was replaced (page 96 - pid 8)
Process 7 accessed page 9 - frame 1
Memory is full! Frame 2 was replaced (page 48 - pid 9)
Process 9 accessed page 16 - frame 2
Memory is full! Frame 3 was replaced (page 43 - pid 8)
Process 8 accessed page 47 - frame 3
Page 69 in memory (frame 0)!
Process 0 accessed page 69 - frame 0
Memory is full! Frame 4 was replaced (page 17 - pid 4)
Process 2 accessed page 99 - frame 4
Memory is full! Frame 5 was replaced (page 75 - pid 7)
Process 11 accessed page 55 - frame 5
Memory is full! Frame 6 was replaced (page 123 - pid 9)
Process 10 accessed page 124 - frame 6
Memory is full! Frame 7 was replaced (page 37 - pid 3)
Process 3 accessed page 46 - frame 7
Memory is full! Frame 0 was replaced (page 69 - pid 0)
Process 10 accessed page 58 - frame 0
Memory is full! Frame 1 was replaced (page 9 - pid 7)
Process 8 accessed page 22 - frame 1
Memory is full! Frame 2 was replaced (page 16 - pid 9)
Process 12 accessed page 91 - frame 2
Memory is full! Frame 3 was replaced (page 47 - pid 8)
Process 4 accessed page 51 - frame 3
Memory is full! Frame 4 was replaced (page 99 - pid 2)
Process 0 accessed page 67 - frame 4
Memory is full! Frame 5 was replaced (page 55 - pid 11)
Process 3 accessed page 36 - frame 5
Memory is full! Frame 6 was replaced (page 124 - pid 10)
Process 10 accessed page 1 - frame 6
Memory is full! Frame 7 was replaced (page 46 - pid 3)
Process 5 accessed page 24 - frame 7
Memory is full! Frame 0 was replaced (page 58 - pid 10)
Process 0 accessed page 122 - frame 0
Memory is full! Frame 1 was replaced (page 22 - pid 8)
Process 6 accessed page 86 - frame 1
Memory is full! Frame 2 was replaced (page 91 - pid 12)
Process 7 accessed page 8 - frame 2
Memory is full! Frame 3 was replaced (page 51 - pid 4)
Process 9 accessed page 104 - frame 3
Memory is full! Frame 4 was replaced (page 67 - pid 0)
Process 10 accessed page 118 - frame 4
Memory is full! Frame 5 was replaced (page 36 - pid 3)
Process 1 accessed page 89 - frame 5
Memory is full! Frame 6 was replaced (page 1 - pid 10)
Process 7 accessed page 107 - frame 6
Memory is full! Frame 7 was replaced (page 24 - pid 5)
Process 4 accessed page 77 - frame 7
Memory is full! Frame 0 was replaced (page 122 - pid 0)
Process 6 accessed page 82 - frame 0
Memory is full! Frame 1 was replaced (page 86 - pid 6)
Process 6 accessed page 90 - frame 1
Memory is full! Frame 2 was replaced (page 8 - pid 7)
Process 11 accessed page 33 - frame 2
Memory is full! Frame 3 was replaced (page 104 - pid 9)
Process 3 accessed page 50 - frame 3
Page 77 in memory (frame 7)!
Process 4 accessed page 77 - frame 7
Memory is full! Frame 4 was replaced (page 118 - pid 10)
Process 6 accessed page 18 - frame 4
Memory is full! Frame 5 was replaced (page 89 - pid 1)
Process 2 accessed page 88 - frame 5
Memory is full! Frame 6 was replaced (page 107 - pid 7)
Process 10 accessed page 124 - frame 6
Memory is full! Frame 7 was replaced (page 77 - pid 4)
Process 9 accessed page 48 - frame 7
Memory is full! Frame 0 was replaced (page 82 - pid 6)
Process 3 accessed page 21 - frame 0
Memory is full! Frame 1 was replaced (page 90 - pid 6)
Process 1 accessed page 101 - frame 1
Memory is full! Frame 2 was replaced (page 33 - pid 11)
Process 9 accessed page 123 - frame 2
Memory is full! Frame 3 was replaced (page 50 - pid 3)
Process 0 accessed page 117 - frame 3
Memory is full! Frame 4 was replaced (page 18 - pid 6)
Process 2 accessed page 56 - frame 4
Memory is full! Frame 5 was replaced (page 88 - pid 2)
Process 6 accessed page 81 - frame 5
Memory is full! Frame 6 was replaced (page 124 - pid 10)
Process 8 accessed page 112 - frame 6
Memory is full! Frame 7 was replaced (page 48 - pid 9)
Process 0 accessed page 122 - frame 7
Memory is full! Frame 0 was replaced (page 21 - pid 3)
Process 12 accessed page 97 - frame 0
Memory is full! Frame 1 was replaced (page 101 - pid 1)
Process 9 accessed page 48 - frame 1
Memory is full! Frame 2 was replaced (page 123 - pid 9)
Process 12 accessed page 6 - frame 2
Memory is full! Frame 3 was replaced (page 117 - pid 0)
Process 7 accessed page 95 - frame 3
Memory is full! Frame 4 was replaced (page 56 - pid 2)
Process 10 accessed page 118 - frame 4
Memory is full! Frame 5 was replaced (page 81 - pid 6)
Process 7 accessed page 59 - frame 5
Memory is full! Frame 6 was replaced (page 112 - pid 8)
Process 3 accessed page 40 - frame 6
Memory is full! Frame 7 was replaced (page 122 - pid 0)
Process 6 accessed page 100 - frame 7
Memory is full! Frame 0 was replaced (page 97 - pid 12)
Process 3 accessed page 50 - frame 0
Page 95 in memory (frame 3)!
Process 7 accessed page 95 - frame 3
Memory is full! Frame 1 was replaced (page 48 - pid 9)
Process 2 accessed page 99 - frame 1
Memory is full! Frame 2 was replaced (page 6 - pid 12)
Process 11 accessed page 33 - frame 2
Memory is full! Frame 3 was replaced (page 95 - pid 7)
Process 12 accessed page 6 - frame 3
Memory is full! Frame 4 was replaced (page 118 - pid 10)
Process 9 accessed page 78 - frame 4
Memory is full! Frame 5 was replaced (page 59 - pid 7)
Process 2 accessed page 25 - frame 5
Memory is full! Frame 6 was replaced (page 40 - pid 3)
Process 6 accessed page 82 - frame 6
Memory is full! Frame 7 was replaced (page 100 - pid 6)
Process 8 accessed page 84 - frame 7
Memory is full! Frame 0 was replaced (page 50 - pid 3)
Process 12 accessed page 93 - frame 0
Memory is full! Frame 1 was replaced (page 99 - pid 2)
Process 11 accessed page 29 - frame 1
Page 82 in memory (frame 6)!
Process 6 accessed page 82 - frame 6
Memory is full! Frame 2 was replaced (page 33 - pid 11)
Process 11 accessed page 39 - frame 2
Memory is full! Frame 3 was replaced (page 6 - pid 12)
Process 1 accessed page 11 - frame 3
Page 29 in memory (frame 1)!
Process 11 accessed page 29 - frame 1
Memory is full! Frame 4 was replaced (page 78 - pid 9)
Process 1 accessed page 101 - frame 4
Memory is full! Frame 5 was replaced (page 25 - pid 2)
Process 6 accessed page 100 - frame 5
Memory is full! Frame 6 was replaced (page 82 - pid 6)
Process 4 accessed page 34 - frame 6
Memory is full! Frame 7 was replaced (page 84 - pid 8)
Process 1 accessed page 79 - frame 7
Memory is full! Frame 0 was replaced (page 93 - pid 12)
Process 9 accessed page 26 - frame 0
Memory is full! Frame 1 was replaced (page 29 - pid 11)
Process 11 accessed page 35 - frame 1
Memory is full! Frame 2 was replaced (page 39 - pid 11)
Process 10 accessed page 108 - frame 2
Memory is full! Frame 3 was replaced (page 11 - pid 1)
Process 5 accessed page 60 - frame 3
Memory is full! Frame 4 was replaced (page 101 - pid 1)
Process 3 accessed page 7 - frame 4
Memory is full! Frame 5 was replaced (page 100 - pid 6)
Process 4 accessed page 119 - frame 5
Memory is full! Frame 6 was replaced (page 34 - pid 4)
Process 6 accessed page 41 - frame 6
Memory is full! Frame 7 was replaced (page 79 - pid 1)
Process 11 accessed page 33 - frame 7
Memory is full! Frame 0 was replaced (page 26 - pid 9)
Process 3 accessed page 44 - frame 0
Memory is full! Frame 1 was replaced (page 35 - pid 11)
Process 1 accessed page 101 - frame 1
Memory is full! Frame 2 was replaced (page 108 - pid 10)
Process 8 accessed page 112 - frame 2
Memory is full! Frame 3 was replaced (page 60 - pid 5)
Process 9 accessed page 48 - frame 3
Memory is full! Frame 4 was replaced (page 7 - pid 3)
Process 2 accessed page 105 - frame 4
Memory is full! Frame 5 was replaced (page 119 - pid 4)
Process 9 accessed page 123 - frame 5
Memory is full! Frame 6 was replaced (page 41 - pid 6)
Process 7 accessed page 75 - frame 6
Memory is full! Frame 7 was replaced (page 33 - pid 11)
Process 2 accessed page 25 - frame 7
Memory is full! Frame 0 was replaced (page 44 - pid 3)
Process 8 accessed page 85 - frame 0
Memory is full! Frame 1 was replaced (page 101 - pid 1)
Process 3 accessed page 13 - frame 1
Memory is full! Frame 2 was replaced (page 112 - pid 8)
Process 0 accessed page 122 - frame 2
Memory is full! Frame 3 was replaced (page 48 - pid 9)
Process 9 accessed page 104 - frame 3
Memory is full! Frame 4 was replaced (page 105 - pid 2)
Process 4 accessed page 116 - frame 4
Memory is full! Frame 5 was replaced (page 123 - pid 9)
Process 5 accessed page 115 - frame 5
Memory is full! Frame 6 was replaced (page 75 - pid 7)
Process 8 accessed page 84 - frame 6
Memory is full! Frame 7 was replaced (page 25 - pid 2)
Process 0 accessed page 68 - frame 7
Memory is full! Frame 0 was replaced (page 85 - pid 8)
Process 8 accessed page 112 - frame 0
Memory is full! Frame 1 was replaced (page 13 - pid 3)
Process 4 accessed page 121 - frame 1
Memory is full! Frame 2 was replaced (page 122 - pid 0)
Process 12 accessed page 20 - frame 2
Memory is full! Frame 3 was replaced (page 104 - pid 9)
Process 3 accessed page 40 - frame 3
Memory is full! Frame 4 was replaced (page 116 - pid 4)
Process 6 accessed page 90 - frame 4
Memory is full! Frame 5 was replaced (page 115 - pid 5)
Process 12 accessed page 91 - frame 5
Memory is full! Frame 6 was replaced (page 84 - pid 8)
Process 12 accessed page 106 - frame 6
Memory is full! Frame 7 was replaced (page 68 - pid 0)
Process 2 accessed page 45 - frame 7
Memory is full! Frame 0 was replaced (page 112 - pid 8)
Process 12 accessed page 57 - frame 0
Memory is full! Frame 1 was replaced (page 121 - pid 4)
Process 7 accessed page 120 - frame 1
Page 91 in memory (frame 5)!
Process 12 accessed page 91 - frame 5
Memory is full! Frame 2 was replaced (page 20 - pid 12)
Process 8 accessed page 85 - frame 2
Page 106 in memory (frame 6)!
Process 12 accessed page 106 - frame 6
Page 40 in memory (frame 3)!
Process 3 accessed page 40 - frame 3
Memory is full! Frame 3 was replaced (page 40 - pid 3)
Process 1 accessed page 28 - frame 3
Memory is full! Frame 4 was replaced (page 90 - pid 6)
Process 1 accessed page 14 - frame 4
Memory is full! Frame 5 was replaced (page 91 - pid 12)
Process 10 accessed page 58 - frame 5
Page 28 in memory (frame 3)!
Process 1 accessed page 28 - frame 3
Memory is full! Frame 6 was replaced (page 106 - pid 12)
Process 7 accessed page 59 - frame 6
Memory is full! Frame 7 was replaced (page 45 - pid 2)
Process 12 accessed page 93 - frame 7
Memory is full! Frame 0 was replaced (page 57 - pid 12)
Process 9 accessed page 26 - frame 0
Memory is full! Frame 1 was replaced (page 120 - pid 7)
Process 1 accessed page 101 - frame 1
Memory is full! Frame 2 was replaced (page 85 - pid 8)
Process 12 accessed page 76 - frame 2
Memory is full! Frame 3 was replaced (page 28 - pid 1)
Process 1 accessed page 89 - frame 3
Memory is full! Frame 4 was replaced (page 14 - pid 1)
Process 10 accessed page 108 - frame 4
Memory is full! Frame 5 was replaced (page 58 - pid 10)
Process 5 accessed page 115 - frame 5
Memory is full! Frame 6 was replaced (page 59 - pid 7)
Process 10 accessed page 124 - frame 6
Memory is full! Frame 7 was replaced (page 93 - pid 12)
Process 12 accessed page 57 - frame 7
Memory is full! Frame 0 was replaced (page 26 - pid 9)
Process 2 accessed page 45 - frame 0
Memory is full! Frame 1 was replaced (page 101 - pid 1)
Process 6 accessed page 49 - frame 1
Memory is full! Frame 2 was replaced (page 76 - pid 12)
Process 11 accessed page 0 - frame 2
Memory is full! Frame 3 was replaced (page 89 - pid 1)
Process 3 accessed page 74 - frame 3
Page 74 in memory (frame 3)!
Process 3 accessed page 74 - frame 3
Memory is full! Frame 4 was replaced (page 108 - pid 10)
Process 8 accessed page 22 - frame 4
Memory is full! Frame 5 was replaced (page 115 - pid 5)
Process 5 accessed page 54 - frame 5
Page 124 in memory (frame 6)!
Process 10 accessed page 124 - frame 6
Memory is full! Frame 6 was replaced (page 124 - pid 10)
Process 11 accessed page 35 - frame 6
Memory is full! Frame 7 was replaced (page 57 - pid 12)
Process 7 accessed page 75 - frame 7
Memory is full! Frame 0 was replaced (page 45 - pid 2)
Process 1 accessed page 89 - frame 0
Memory is full! Frame 1 was replaced (page 49 - pid 6)
Process 1 accessed page 73 - frame 1
Memory is full! Frame 2 was replaced (page 0 - pid 11)
Process 10 accessed page 1 - frame 2
Memory is full! Frame 3 was replaced (page 74 - pid 3)
Process 5 accessed page 60 - frame 3
Memory is full! Frame 4 was replaced (page 22 - pid 8)
Process 4 accessed page 34 - frame 4
Memory is full! Frame 5 was replaced (page 54 - pid 5)
Process 5 accessed page 65 - frame 5
Memory is full! Frame 6 was replaced (page 35 - pid 11)
Process 7 accessed page 8 - frame 6
Memory is full! Frame 7 was replaced (page 75 - pid 7)
Process 0 accessed page 61 - frame 7
Memory is full! Frame 0 was replaced (page 89 - pid 1)
Process 10 accessed page 58 - frame 0
Memory is full! Frame 1 was replaced (page 73 - pid 1)
Process 0 accessed page 122 - frame 1
Memory is full! Frame 2 was replaced (page 1 - pid 10)
Process 12 accessed page 97 - frame 2
Memory is full! Frame 3 was replaced (page 60 - pid 5)
Process 1 accessed page 79 - frame 3
Memory is full! Frame 4 was replaced (page 34 - pid 4)
Process 4 accessed page 5 - frame 4
Memory is full! Frame 5 was replaced (page 65 - pid 5)
Process 1 accessed page 73 - frame 5
Memory is full! Frame 6 was replaced (page 8 - pid 7)
Process 8 accessed page 112 - frame 6
Memory is full! Frame 7 was replaced (page 61 - pid 0)
Process 1 accessed page 98 - frame 7
Memory is full! Frame 0 was replaced (page 58 - pid 10)
Process 10 accessed page 1 - frame 0
Memory is full! Frame 1 was replaced (page 122 - pid 0)
Process 10 accessed page 108 - frame 1
Memory is full! Frame 2 was replaced (page 97 - pid 12)
Process 12 accessed page 91 - frame 2
Memory is full! Frame 3 was replaced (page 79 - pid 1)
Process 7 accessed page 120 - frame 3
Memory is full! Frame 4 was replaced (page 5 - pid 4)
Process 6 accessed page 18 - frame 4
Memory is full! Frame 5 was replaced (page 73 - pid 1)
Process 11 accessed page 33 - frame 5
Page 91 in memory (frame 2)!
Process 12 accessed page 91 - frame 2
Memory is full! Frame 6 was replaced (page 112 - pid 8)
Process 0 accessed page 2 - frame 6
Memory is full! Frame 7 was replaced (page 98 - pid 1)
Process 7 accessed page 59 - frame 7
Memory is full! Frame 0 was replaced (page 1 - pid 10)
Process 0 accessed page 15 - frame 0
Page 15 in memory (frame 0)!
Process 0 accessed page 15 - frame 0
Memory is full! Frame 1 was replaced (page 108 - pid 10)
Process 1 accessed page 14 - frame 1
Memory is full! Frame 2 was replaced (page 91 - pid 12)
Process 5 accessed page 54 - frame 2
Memory is full! Frame 3 was replaced (page 120 - pid 7)
Process 10 accessed page 118 - frame 3
Memory is full! Frame 4 was replaced (page 18 - pid 6)
Process 11 accessed page 29 - frame 4
Page 29 in memory (frame 4)!
Process 11 accessed page 29 - frame 4
Memory is full! Frame 5 was replaced (page 33 - pid 11)
Process 7 accessed page 95 - frame 5
Memory is full! Frame 6 was replaced (page 2 - pid 0)
Process 9 accessed page 26 - frame 6
Memory is full! Frame 7 was replaced (page 59 - pid 7)
Process 6 accessed page 86 - frame 7
Memory is full! Frame 0 was replaced (page 15 - pid 0)
Process 8 accessed page 126 - frame 0
Memory is full! Frame 1 was replaced (page 14 - pid 1)
Process 3 accessed page 72 - frame 1
Memory is full! Frame 2 was replaced (page 54 - pid 5)
Process 12 accessed page 91 - frame 2
Memory is full! Frame 3 was replaced (page 118 - pid 10)
Process 3 accessed page 13 - frame 3
Memory is full! Frame 4 was replaced (page 29 - pid 11)
Process 0 accessed page 117 - frame 4
Memory is full! Frame 5 was replaced (page 95 - pid 7)
Process 8 accessed page 22 - frame 5
Memory is full! Frame 6 was replaced (page 26 - pid 9)
Process 7 accessed page 95 - frame 6
Memory is full! Frame 7 was replaced (page 86 - pid 6)
Process 7 accessed page 120 - frame 7
Memory is full! Frame 0 was replaced (page 126 - pid 8)
Process 5 accessed page 53 - frame 0
Memory is full! Frame 1 was replaced (page 72 - pid 3)
Process 11 accessed page 55 - frame 1
Memory is full! Frame 2 was replaced (page 91 - pid 12)
Process 6 accessed page 82 - frame 2
Memory is full! Frame 3 was replaced (page 13 - pid 3)
Process 12 accessed page 76 - frame 3
Memory is full! Frame 4 was replaced (page 117 - pid 0)
Process 6 accessed page 32 - frame 4
Memory is full! Frame 5 was replaced (page 22 - pid 8)
Process 10 accessed page 118 - frame 5
Memory is full! Frame 6 was replaced (page 95 - pid 7)
Process 3 accessed page 7 - frame 6
Memory is full! Frame 7 was replaced (page 120 - pid 7)
Process 3 accessed page 114 - frame 7
Memory is full! Frame 0 was replaced (page 53 - pid 5)
Process 12 accessed page 93 - frame 0
Memory is full! Frame 1 was replaced (page 55 - pid 11)
Process 9 accessed page 123 - frame 1
Memory is full! Frame 2 was replaced (page 82 - pid 6)
Process 4 accessed page 34 - frame 2
Memory is full! Frame 3 was replaced (page 76 - pid 12)
Process 7 accessed page 83 - frame 3
Memory is full! Frame 4 was replaced (page 32 - pid 6)
Process 7 accessed page 8 - frame 4
Memory is full! Frame 5 was replaced (page 118 - pid 10)
Process 12 accessed page 20 - frame 5
Memory is full! Frame 6 was replaced (page 7 - pid 3)
Process 12 accessed page 76 - frame 6
Memory is full! Frame 7 was replaced (page 114 - pid 3)
Process 3 accessed page 74 - frame 7
Memory is full! Frame 0 was replaced (page 93 - pid 12)
Process 0 accessed page 122 - frame 0
Memory is full! Frame 1 was replaced (page 123 - pid 9)
Process 9 accessed page 92 - frame 1
Memory is full! Frame 2 was replaced (page 34 - pid 4)
Process 10 accessed page 23 - frame 2
Memory is full! Frame 3 was replaced (page 83 - pid 7)
Process 1 accessed page 101 - frame 3
Memory is full! Frame 4 was replaced (page 8 - pid 7)
Process 1 accessed page 11 - frame 4
Memory is full! Frame 5 was replaced (page 20 - pid 12)
Process 11 accessed page 94 - frame 5
Memory is full! Frame 6 was replaced (page 76 - pid 12)
Process 5 accessed page 24 - frame 6
Memory is full! Frame 7 was replaced (page 74 - pid 3)
Process 0 accessed page 68 - frame 7
Memory is full! Frame 0 was replaced (page 122 - pid 0)
Process 5 accessed page 115 - frame 0
Memory is full! Frame 1 was replaced (page 92 - pid 9)
Process 11 accessed page 29 - frame 1
Memory is full! Frame 2 was replaced (page 23 - pid 10)
Process 2 accessed page 45 - frame 2
Memory is full! Frame 3 was replaced (page 101 - pid 1)
Process 10 accessed page 118 - frame 3
Memory is full! Frame 4 was replaced (page 11 - pid 1)
Process 4 accessed page 17 - frame 4
